src/plugins/orangehrmRecruitmentPlugin/config/routes.yaml

```diff
@@ -182,6 +182,21 @@ apiv2_recrutiment_candidate_history:
     _api: OrangeHRM\Recruitment\Api\CandidateHistoryAPI
     _key: historyId
 
+apiv2_recruitment_vacancy_list:
+  path: /api/v2/recruitment/public/vacancies
+  controller: OrangeHRM\Recruitment\Controller\VacancyListRestController::handle
+  methods: [ GET ]
+
+apiv2_recruitment_vacancy_by_id:
+  path: /api/v2/recruitment/public/vacancies/{id}
+  controller: OrangeHRM\Recruitment\Controller\VacancyRestController::handle
+  methods: [ GET, POST ]
+
+apiv2_recruitment_applicants:
+  path: /recruitment/public/applicants
+  controller: OrangeHRM\Recruitment\Controller\ApplicantController::handle
+  methods: [ GET, POST ]
```

Comment:
this is not a apiv2
also are we handle `GET` in `ApplicantController` :thinking:

Reply:
removed GET, renamed to recruitment_public_applicants